app/screens/scheduled_post_options/scheduled_post_picker.tsx

```diff
@@ -75,10 +79,27 @@ export function ScheduledPostOptions({currentUserTimezone}: Props) {
         setSelectedTime(selectedValue);
     }, []);
 
-    const onSchedule = useCallback(async () => {
-        // TODO - to be implemented later
-        logInfo('Schedule post', selectedTime);
-    }, [selectedTime]);
+    const handleOnSchedule = usePreventDoubleTap(useCallback(async () => {
+        if (!selectedTime) {
+            logInfo('ScheduledPostOptions', 'No time selected');
```

Comment:
Do we have this log because this should never happen? If so, shouldn't it be debug level? If not, shouldn't we show something to the user?

Reply:
Moved it to debug level.